### PR TITLE
Ignore unknown cotexts for nmstate/kubernetes-nmstate

### PR DESCRIPTION
**Pull request by fgimenez**
Will prevent tide being blocked because of gh actions errors

/cc @qinqon @dhiller 

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml [hunk 1]
```diff
@@ -186,6 +186,10 @@ tide:
         dashboard:
           skip-unknown-contexts: true
           from-branch-protection: true
+     nmstate:
+      repos:
```
Comment (qinqon) [inline github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml:190]: It has bad identation, it's missing an space.

**Review by qinqon — CHANGES_REQUESTED** [review]

File: github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml (hunk 1 above)
Comment (fgimenez) [reply github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml]: thanks for the heads up! it is not required for the context_options to be applied but good to have it fixed, already pushed the changes

**Review by dhiller — APPROVED** [review]
/approve

**Pull request merged**